### PR TITLE
Added the Art of Memory Forensics

### PR DESCRIPTION
**Pull request by Satharus**
A must read for DFIR personnel.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,6 +82,7 @@ Digital Forensics and Incident Response (DFIR) teams are groups of people in an
 * [Incident Response with Threat Intelligence](https://www.amazon.com/Incident-response-Threat-Intelligence-intelligence-based/dp/1801072957) - Great reference to build an incident response plan based also on Threat Intelligence. By Roberto Martinez.
 * [Intelligence-Driven Incident Response](https://www.amazon.com/Intelligence-Driven-Incident-Response-Outwitting-Adversary-ebook-dp-B074ZRN5T7/dp/B074ZRN5T7) - By Scott J. Roberts, Rebekah Brown.
 * [Operator Handbook: Red Team + OSINT + Blue Team Reference](https://www.amazon.com/Operator-Handbook-Team-OSINT-Reference/dp/B085RR67H5/) - Great reference for incident responders.
+* [The Art of Memory Forensics](https://www.amazon.com/Art-Memory-Forensics-Detecting-Malware/dp/1118825098/) - Detecting Malware and Threats in Windows, Linux, and Mac Memory. By  Michael Ligh, Andrew Case, Jamie Levy, and AAron Walters.
```
Comment (meirwah) [inline README.md:85]: plz note the list is ordered by alphabetical order, plz place in correct place

Comment (Satharus) [reply README.md]: Done!

File: README.md [hunk 2]
```diff
@@ -74,6 +74,7 @@ Digital Forensics and Incident Response (DFIR) teams are groups of people in an
 ### Books
 
 * [Applied Incident Response](https://www.amazon.com/Applied-Incident-Response-Steve-Anson/dp/1119560268/) - Steve Anson's book on Incident Response.
+* [The Art of Memory Forensics](https://www.amazon.com/Art-Memory-Forensics-Detecting-Malware/dp/1118825098/) - Detecting Malware and Threats in Windows, Linux, and Mac Memory. By  Michael Ligh, Andrew Case, Jamie Levy, and AAron Walters.
```
Comment (meirwah) [inline README.md:77]: just drop the `The ` part plz

Comment (Satharus) [reply README.md]: Done!

**Comment by meirwah** [comment]
seems spell check fails on author name `Michael Ligh`:  
image

maybe remove  authors names to fix that ?
the names appear in link anyway

**Comment by Satharus** [comment]
> seems spell check fails on author name `Michael Ligh`: image
> 
> maybe remove authors names to fix that ? the names appear in link anyway

Done. Removed the names of the 4 authors.

**Review by meirwah — APPROVED** [review]

**Pull request merged**